scripts/webpack/prod.client.config.js

```diff
@@ -120,6 +120,8 @@ module.exports = ({isDeploy, isStats}) => ({
       __PRODUCTION__: true,
       __APP_VERSION__: JSON.stringify(process.env.npm_package_version),
       'process.env.DEBUG': false,
+      'process.env.NODE_ENV': JSON.stringify('production'),
+      'process.env.PROTOO_LISTEN_PORT': JSON.stringify((process.env.PROTOO_LISTEN_PORT || 4444) - 1),
```

Comment:
-1 can remove NODE_ENV, it's handled by webpack config.
-1 protoo is no more

Reply:
Oops my bad! Accidentally included the PROTOO\_LISTEN\_PORT during merge conflict resolution. The NODE\_ENV though is in [master branch](https://github.com/ParabolInc/parabol/blob/master/scripts/webpack/prod.client.config.js#L123) today.